mdm/api/src/main/java/com/gentics/mesh/core/data/dao/ContainerDao.java

```diff
@@ -48,6 +48,12 @@
 	 */
 	Result<? extends HibBranch> getBranches(SCV version);
 
+	/**
+	 * Method called before version is deleted
+	 * @param version
```

Comment:
This should be moved to PersistingContainerDao, since API should not know about the database.
ps: it's me, @plyhun , from the alter ego acount.

Reply:
Makes sense, will change it